src/infrastructure/group-store/dynamodb-group-store.test.ts

```diff
@@ -50,6 +50,17 @@ describe("test groups stores", () => {
     const nameHash3 = keccak256(toUtf8Bytes(savedGroup3.name + "/" + "2"));
     const savedId3 = BigNumber.from(nameHash3).mod(UINT128_MAX).toHexString();
     expect(savedGroup3.id).toBe(savedId3);
+
+    const savedGroup4 = await dynamodbGroupStore.save(testGroups.group1_2);
```

Comment:
What is the aim of this test?
There should be a check after the update correct? Why not update the displayName?

Reply:
The purpose of this test is to make sure that a group with a displayName field can be saved &updated in the dynamoDB without any errors. But I agree, for the update we can also update the displayName and check that all field has been successfully updated  👍